### ISSUE TITLE
Crash on "Singleplayer" with no worlds

### ISSUE THREAD

**Issue by Jerozgen** [Antikyth/searchable#25]
The "Singleplayer" button redirects to the world creation screen if there's no worlds. This causes a crash

### Mods
Searchable 0.2.1+1.20.1
Quilted Fabric API 7.3.0+0.89.0-1.20.1
MixinTrace 1.1.1+1.17

### Stacktrace
```
java.lang.NullPointerException: Cannot assign field "field_22763" because "this.field_3224" is null
	at net.minecraft.class_526.method_19940(class_526.java:122)
	at net.minecraft.class_528.method_20157(class_528.java:223)
	at net.minecraft.class_528.method_25313(class_528.java:63)
	at net.minecraft.class_528.handler$zio000$searchable$onFilter(class_528.java:574)
	at net.minecraft.class_528.method_43454(class_528.java)
	at net.minecraft.class_528.method_44678(class_528.java:146)
	at net.minecraft.class_528.<init>(class_528.java:97)
	at net.minecraft.class_526.method_25426(class_526.java:54)
	at net.minecraft.class_437.method_25423(class_437.java:297)
	at net.minecraft.class_310.method_1507(class_310.java:1080)
	at net.minecraft.class_442.method_19861(class_442.java:159)
	at net.minecraft.class_4185.method_25306(class_4185.java:94)
	at net.minecraft.class_4264.method_25348(class_4264.java:56)
	at net.minecraft.class_339.method_25402(class_339.java:189)
	at net.minecraft.class_4069.method_25402(class_4069.java:38)
	at net.minecraft.class_442.method_25402(class_442.java:299)
	at net.minecraft.class_312.method_1611(class_312.java:98)
	at net.minecraft.class_437.method_25412(class_437.java:409)
	at net.minecraft.class_312.method_1601(class_312.java:98)
	at net.minecraft.class_312.method_22686(class_312.java:169)
	at net.minecraft.class_1255.execute(class_1255.java:102)
	at net.minecraft.class_312.method_22684(class_312.java:169)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:237)
	at net.minecraft.class_310.method_1523(class_310.java:1244)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
```

**Comment by Antikyth** [comment]
Thanks for this report.  I'm really swamped at uni at the moment so I'm not sure when I will be able to address it unfortunately (probably not for between 2 and 4 weeks from now unless I find some time on a weekend), but just commenting to say it is planned.

**Comment by Antikyth** [comment]
Fix released in 0.2.2.
